static/docs/get-started/example-pipeline.md

```diff
@@ -51,7 +51,7 @@ your global packages clean and untouched:
 ```dvc
     $ virtualenv .env
     $ source .env/bin/activate
-    $ echo '.env/' >> .gitignore
+    $ echo ".env/" >> .gitignore
```

Comment:
Just in case also applied to other commands (not just `git commit -m`), like the one above.